dynamics_class: "dubins_car"
safe_set:
  lo: 
    - -5.0
    - -5.0
    - -3.1416
  hi: 
    - 5.0
    - 5.0
    - 3.1416

obstacles: 
  
  obstacle1:
    type: 'Circle'
    mode: 'Detection'
    indices:
      - 0
      - 1
    center: 
      - 1.0
      - 0.0
    radius: .1
    padding: .1
    detectionradius: .5
    detectionrateHZ: 20

  obstacle2:
    type: 'Circle'
    mode: 'Detection'
    indices:
      - 0
      - 1
    center:
      x: 0.0
      y: 1.0
    radius: .1
    padding: .1
    detectionradius: .5
    detectionrateHZ: 20

  obstacle3:
    type: 'Circle'
    mode: 'Detection'
    indices:
      - 0
      - 1
    center:
      x: -1.0
      y: 0.0
    radius: .1
    padding: .1
    detectionradius: .5
    detectionrateHZ: 20

  obstacle4:
    type: 'circle'
    mode: 'Time'
    indices:
      - 0
      - 1
    center:
      x: 0.0
      y: -1.0
    radius: .1
    padding: .1
    updateTime: 5
    detectionrateHZ: 20

boundary:
  padding: 0
  indices:
    - 0
    - 1
  minVal:
    - -5.0
    - -5.0
  maxVal:
    - 10.0
    - 10.0


state_domain:
  lo: 
    - -5.0
    - -5.0
    - -3.1416
  hi: 
    - 10.0
    - 10.0
    - 3.1416
  resolution:
    - 41
    - 41
    - 41
  periodic_dims: [2]

control_space:
  n_dims: 2
  lo:
    - -1.0
    - 0.0
  hi: 
    - 1.0
    - 0.2

disturbance_space:
  n_dims: 0
  lo: None
  hi: None
